src/test/java/net/sf/jsqlparser/statement/builder/JSQLParserFluentModelTests.java

```diff
@@ -38,14 +40,15 @@
     @Test
     public void testParseAndBuild() throws JSQLParserException {
         String statement = "SELECT * FROM tab1 AS t1 " //
-                + "JOIN tab2 t2 ON t1.ref = t2.id WHERE (t1.col1 = ? OR t1.col2 = ?) AND t1.col3 IN ('A')";
+                + "JOIN tab2 t2 ON t1.ref = t2.id WHERE (t1.col1 = ? OR t1.col2 = ?) AND t1.col3 IN ('A') XOR t1.col4";
```

Comment:
You should not adapt tests but create new ones.

Reply:
I've fixed this by introducing a separate test case for the XOR case, restoring the original one to the previous state.